app/controllers/concerns/sessions_management.rb

```diff
@@ -33,7 +33,7 @@ def login(interactor_call)
     if interactor_call.success?
       sign_in_and_redirect(interactor_call.user)
     else
-      error_message(title: t(".#{interactor_call.message}.title"), description: t(".#{interactor_call.message}.description"))
+      send(extract_flash_kind(interactor_call.message), title: t(".#{interactor_call.message}.title"), description: t(".#{interactor_call.message}.description"))
```

Comment:
Est ce possible de modifier le message d'erreur de

> Utilisateur non trouvé

à

> Aucun compte associé à l'adresse email %{emailt}

(Ça serait aussi une opportunité pour faire de l'écriture épicène.)


J'ai voulu essayer de faire une proposition sous forme d'une PR mais je suis bloqué. J'ai explicité mon blocage dans une issue dédiée.

https://github.com/etalab/admin_api_entreprise/issues/1128

Reply:
Tu peux faire une PR en modifiant le texte mentionné dans ce commentaire: https://github.com/etalab/admin_api_entreprise/pull/1126#issuecomment-1632018242 (pas besoin de faire tourner l'app ici)

Reply:
Ah désolé, j'avais pas vu les commentaires de @DorineLam, ravi de voir qu'on est aligné.

J'ose pas faire de PR si je ne peux pas faire de test en local et comme je vois que vous êtes sur le coup, y compris sur la reformulation du message, j'ai l'impression que _my job is done here_. :sweat_smile:

Reply:
Ouais mais on n'a pas les mêmes bases en Ruby ni sur l'application donc les proba sont très différentes quand je code un truc et quand tu en codes un.

Reply:
C'est du texte et non du code ;)

Reply:
(clairement que je ne te demande pas d'écrire du code)